spec/features/skills_form_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+
+describe 'Skill Form', type: :system, js:true do
+  before do
+    visit skills_path
+    click_link(href: new_skill_path)
```

Comment:
Why do you have to set the href?

Reply:
Cause it's likely we change the name of the button but the href stays consistent